core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -1460,9 +1458,13 @@ class AdminClientIntegrationTest extends IntegrationTestHarness with Logging {
     val partition2 = new TopicPartition("elect-preferred-leaders-topic-2", 0)
     TestUtils.createTopic(zkClient, partition2.topic, Map[Int, Seq[Int]](partition2.partition -> prefer0), servers)
 
-    def preferredLeader(topicPartition: TopicPartition) =
-      client.describeTopics(asList(topicPartition.topic)).values.get(topicPartition.topic).
-        get.partitions.get(topicPartition.partition).replicas.get(0).id
+    def preferredLeader(topicPartition: TopicPartition): Int =
+      getTopicMetadata(client, topicPartition.topic)
+        .partitions
+        .get(topicPartition.partition)
+        .replicas
+        .get(0)
+        .id
```

Comment:
:+1:

Reply:
This style is a bit odd. Writing vertically instead of horizontally. :)

Reply:
Heh, I find it harder to read when so many calls are squashed together on one line

Reply:
When you have lambdas, sure. But when they're just chained calls, having one per line is a bit excessive. Also, if there are so many chained calls, you can introduce variables to make it more readable.